sdk/eventhub/azure-eventhubs/azure/eventhub/__init__.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-__version__ = "1.3.2"
+__version__ = "1.3.3b1"
```

Comment:
I'm guessing we should revert these version changes in favour of those made in the other PR?

Reply:
Apt timing, I was about to start a discussion with you about how we want to handle this going forward.  I'm open to any preferences you have one way or the other, as you see I was somewhat ambivalent as to which we settled on :P